stanford_profile.install

```diff
@@ -22,8 +22,9 @@ function stanford_profile_install() {
 
   // Assign user 1 the "administrator" role.
   $user = User::load(1);
-  $user->name = 'sws-developers';
-  $user->roles[] = 'administrator';
+  $user->set('name', 'sws-developers');
+  $user->set('mail', 'sws-developers@lists.stanford.edu');
```

Comment:
I am good with hard coding this. I don't think we will ever want to hand over user 1 to any of our clients.

Reply:
thing to note. `blt drupal:install` overrides these. but `drush si config_installer` uses them. I think when installing through ACSF UI it will work correctly